src/components/SliderProgress.vue

```diff
@@ -0,0 +1,206 @@
+<script lang="ts">
+import { defineComponent, computed } from 'vue';
+import { Screen } from 'quasar';
+import { Navigation, A11y } from 'swiper/modules';
+
+// components
+import { Swiper, SwiperSlide } from 'swiper/vue';
+import CardProgressSlider from './CardProgressSlider.vue';
+
+// types
+import { CardProgress, Link, ItemStatistics } from './types';
+
+export default defineComponent({
+  name: 'SliderProgress',
+  components: {
+    Swiper,
+    SwiperSlide,
+    CardProgressSlider,
+  },
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+    stats: {
+      type: Array as () => ItemStatistics[],
+    },
+    cards: {
+      type: Array as () => CardProgress[],
+      required: true,
+    },
+    button: {
+      type: Object as () => Link,
+      required: false,
+    },
+  },
+  setup() {
+    const isLargeScreen = computed(() => {
+      return Screen.gt.sm;
+    });
+
+    const buttonWidth = computed(() => {
+      return isLargeScreen.value ? 'auto' : '100%';
+    });
+
+    return {
+      buttonWidth,
+      modules: [Navigation, A11y],
+    };
+  },
+});
+</script>
+
+<template>
+  <!-- Component displaying a slider with progress cards -->
+  <!-- Internal Figma link: https://www.figma.com/file/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?type=design&node-id=4858%3A103888&mode=design&t=UUK9mlZ2h5xWmQ1F-1 -->
+  <div class="progress-slider relative-position" data-cy="progress-slider">
+    <div class="row q-col-gutter-lg">
+      <!-- Title -->
+      <h2 class="col-sm-5 text-h6 q-my-none" data-cy="progress-slider-title">
+        {{ title }}
+      </h2>
+      <!-- List of statistics -->
+      <q-list
+        class="col-sm-7 flex flex-wrap items-center justify-end q-pr-md gap-x-40"
+      >
+        <q-item
+          v-for="item in stats"
+          :key="item.icon"
+          data-cy="progress-slider-stats-item"
+          class="text-grey-10 q-p-none"
+        >
+          <!-- Icon -->
+          <q-icon :name="item.icon" color="blue-grey-3" size="18px" />&nbsp;
+          <!-- Value -->
+          <strong>{{ item.value }}</strong
+          >&nbsp;
+          <!-- Label -->
+          <span>{{ item.label }}</span>
+        </q-item>
+      </q-list>
+    </div>
+    <swiper
```

Comment:
According Vue Swipper [doc](https://swiperjs.com/vue): _Swiper Vue components will likely to be removed in future versions. It is recommended to migrate to [Swiper Element](https://swiperjs.com/element) instead_. 

Your Swipper overriding style looks like a little bit complicated. Is there another simpler way how to do it?

https://stackoverflow.com/questions/65590148/swiperjs-how-do-you-style-the-pagination-bullets

If this Swipper component style is same as in `SliderProgress` componet make style declaration in one place.

Reply:
Updated to Swiper Element. Refactored styles to swiper.scss